.github/workflows/build.yaml

```diff
@@ -2,7 +2,8 @@ name: build
 on:
   pull_request:
   push:
-    branches: [ main ]
+    branches: [ rolling ]
+  workflow_dispatch:
```

Comment:
Do we still need the `workflow_dispatch` key?

Reply:
As we are targeting master for the zenoh dependency I found it useful to be able to run it via `workflow_dispatch` to verify that everything works as expected when a new commit is pushed in zenoh-c .

Reply:
> As we are targeting master for the zenoh dependency I found it useful to be able to run it via `workflow_dispatch` to verify that everything works as expected when a new commit is pushed in zenoh-c .

I'm fine with it for now, though I expect very soon that we should pin the zenoh-c commit that we are using, and only update it when we need a new feature.